app/src/main/java/com/example/playlistmaker/FormatTrack.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.playlistmaker
+
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class FormatTrack {
+    companion object {
```

Comment:
Можно сделать просто вот так:
```suggestion
object FormatTrack {
```

Reply:
Да, так еще красивее)